### PR TITLE
Additional lazy init

### PR DESCRIPTION
**Pull request by ciliakas**
Added a few lazy initiliazations to hopefully better fulfill the requirement

### CODE REVIEW

File: src/Utilities/EmailSender.cs [hunk 1]
```diff
@@ -8,6 +8,7 @@ namespace ePiggyWeb.Utilities
 {
     public class EmailSender
     {
+        public const string Email = "Email";
```
Comment (Stokalas) [inline src/Utilities/EmailSender.cs:11]: Where is this used?

File: src/Pages/ForgotPassword.cshtml.cs [hunk 2]
```diff
@@ -52,7 +55,9 @@ public async Task<IActionResult> OnGet()
                 return Page();
             }
             Email = Request.Cookies["Email"];
-            var recoveryCode = (await EmailSender.SendRecoveryCodeAsync(Email)).ToString();
+
+            Configuration.GetSection(Utilities.EmailSender.Email).Bind(EmailSender.Value);
```
Comment (Stokalas) [inline src/Pages/ForgotPassword.cshtml.cs:59]: Can you explain what this line does?

**Review by Stokalas — APPROVED** [review]
Nothing much to say, since as I understand you simply added lazy init to already existing code. Just twu questions

File: src/Utilities/EmailSender.cs (hunk 1 above)
Comment (ciliakas) [reply src/Utilities/EmailSender.cs:11]: This is used in the place where you asked your other question, this string is used to access the data from app settings through configuration

File: src/Pages/ForgotPassword.cshtml.cs (hunk 2 above)
Comment (ciliakas) [reply src/Pages/ForgotPassword.cshtml.cs:59]: So I'm trying out options pattern a bit more, and what this does is takes a section from app settings (GetSection(Utilities.EmailSender.Email)) where the section name is "Email" (from the const string in above question) and then binds it to the properties of the EmailSender object.
Basically this gets data from app settings and injects it into a object

**Pull request merged**